include/lsst/meas/base/PsfFlux.h

```diff
@@ -90,6 +90,8 @@ class PsfFluxAlgorithm : public SimpleAlgorithm {
     Control _ctrl;
     FluxResultKey _instFluxResultKey;
     afw::table::Key<float> _areaKey;
+    afw::table::Key<float> _chi2Key;
+    afw::table::Key<int> _npixelsKey;
```

Comment:
What's the difference between `_npixelsKey` and `_areaKey`? Consider adding (short) comments to these fields; the only context I have is that this is a `PsfFluxAlgorithm`.

Reply:
These keys get doc fields assigned in the `PsfFluxAlgorithm` constructor; I'll copy those docs to comments here.